### ISSUE TITLE
SimpleSynth becomes silent after repeated triggerAttackReleases (Firefox only)

### ISSUE THREAD

**Issue by mcclure** [Tonejs/Tone.js#117]
[Here is a demo program](http://vote.runhello.com/j/dryad-proto/tone-pop-demo/):
[Here is its source code](https://bitbucket.org/runhello/dryad-artifact/src/3f87fc62bf5368b66f97b9f25c4555ddef28afd4) and [here is the main source file](https://bitbucket.org/runhello/dryad-artifact/src/3f87fc62bf5368b66f97b9f25c4555ddef28afd4/src/app.tsx?at=default&fileviewer=file-view-default) (commit 3f87fc62b)
The test program is using tone 0.6.1 from the website. I test on Chrome 48.0.2564.97 and Firefox 43.0.4 on Mac OS X 10.6.8. (I upgraded to Firefox 44.0 after writing this bug and there were no changes in behavior.)
Click "type here" and type. Each keypress issues a triggerAttackRelease on a PolySynth with 2-polyphony and two SimpleSynths.

Try placing your fingers on the "qwe" keys and just mash them over and over. (Do not use capital letters.) When I do this, I find as I press the keys the tones are becoming quieter and quieter. After five or ten seconds the output is inaudible. This happens ONLY on Firefox, not on Chrome.

I captured the output of my sound card. Please see [http://vote.runhello.com/j/dryad-proto/tone-bug-report/](http://vote.runhello.com/j/dryad-proto/tone-bug-report/), "quietly-firefox" and "pops-2-firefox". Included is a FLAC file and a screenshot showing that at a point where you cannot hear anything in the playback there are still extremely quiet waveforms.

**Comment by tambien** [comment]
I played around with this for a little while and I was able to recreate this behavior. i noticed that if i stop button mashing for a moment (and let the release finish), then it returns to normal volume. So this probably has to do with retriggering the envelope. Though, I'm not sure why it only becomes obvious after about 5 seconds.

I've done a little work on envelope retriggering over the past couple days and i'll see if i can write a test which validates this and passes on FF.
